heartbeat/Filesystem

```diff
@@ -389,6 +389,82 @@ fstype_supported()
 	return $OCF_ERR_INSTALLED
 }
 
+#
+# Determine backend filesystem support state
+#
+backend_support()
+{
+	case "$FSYPE" in
+		overlay)
+			# it should fail immediately if kernel module not found
+			modinfo overlay >/dev/null 2>&1
+			;;
+		overlayfs)
+			# it should fail immediately if kernel module not found
+			modinfo overlayfs >/dev/null 2>&1
+			;;
+		nfs4|nfs|smbfs|cifs|gfs2|ocfs2)
+			# FIXME: how to determine if these service is available?
+			;;
+		*)
+			# any other?
+			;;
+	esac
+	[ $? -eq 0 ] && return 0
```

Comment:
umm, another bug here,
when it run into ` *) ` case, ` $? ` will be 0

will be fixed in next commit